dm/unit/unit.go

```diff
@@ -146,6 +147,35 @@ func IsResumableError(err *pb.ProcessError) bool {
 	return true
 }
 
+// IsResumableDBError checks whether the error is resumable DB error.
+// this is a simplified version of IsResumableError.
+// we use a blacklist to filter out some errors which can not be resumed,
+// all other errors is resumable.
+func IsResumableDBError(err error) bool {
+	if err == nil {
+		return true
+	}
+
+	err = errors.Cause(err)
+	if err == context.Canceled {
+		return false
+	}
+
```

Comment:
as its name, should we check it's a DB error? like a error instance from mysql.MySQLError